homeassistant/components/heos/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
+    comment: Integration is a local push integration
+  brands: done
+  common-modules: todo
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: todo
+  test-before-setup: done
+  unique-config-entry:
+    status: todo
+    comment: |
+      The HEOS integration only supports a single config entry, but needs to be migrated to use
+      the `single_config_entry` flag. HEOS devices interconnect to each other, so connecting to
+      a single node yields access to all the devices setup with HEOS on your network. The HEOS API
+      documentation does not recommend connecting to multiple nodes which would provide no bennefit.
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: done
+    comment: |
+      The integration doesn't provide any additional configuration parameters.
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable:
+    status: todo
+    comment: |
+      The integration currently spams the logs until reconnected
+  parallel-updates:
+    status: todo
+    comment: Needs to be set to 0. The underlying library handles parallel updates.
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration doesn't require re-authentication.
+  test-coverage: done
```

Comment:
In `test_async_setup_entry_loads_platforms` you patch 
```py
with patch.object(
        hass.config_entries, "async_forward_entry_setups"
    ) as forward_mock:
```
I would strongly recommend not to patch internals and just assert that everything is setup by checking the mock that everything has been called that we expect, and that the state of the entry is what we expect and the entities has the state that we expect

Reply:
Comment added to implement this.